packages/perseus/src/widgets/numeric-input/input-with-examples.tsx

```diff
@@ -95,21 +95,20 @@ const InputWithExamples = forwardRef<
         const ariaId = `aria-for-${id}`;
 
         // Generate the provided examples in simple language for screen readers.
-        // If all examples are provided, do not provide them to the screen reader.
-        const examplesAria =
-            props.examples.length === 0
-                ? ""
-                : `${props.examples[0]}
+        const examplesAria = props.shouldShowExamples
```

Comment:
We already have this variable that accomplished the same thing earlier, so we might as well use it! :)